docs/ug/configFiles.md

```diff
@@ -40,11 +40,18 @@ Given below are the details of the various config files used by RepoSense.
 | Ignore Commits List<sup>*+</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash. Additionally, a range of commits can be specified using the `..` notation e.g. `abc123..def456` (both inclusive). |
 | Ignore Authors List<sup>*+</sup> | The list of authors to ignore during analysis. Authors should be specified by their [Git Author Name](#a-note-about-git-author-name). |
 | Shallow Cloning | Enter **`yes`** to clone the repository using Git's shallow cloning functionality. This option can significantly reduce the time taken to clone large repositories. However, the option should ideally be disabled for smaller repositories where the `.git` file is smaller than 500 MB, as it would create overhead. |
+| File Size Limit<sup>+</sup>  | Enter a file size limit for the repository that will override the default file size limit. |
```

Comment:
Should we implement a custom file size limit flag? Most of the columns here have cli flag equivalents.

Reply:
This could make sense! The reason I didn't also add a cli flag for this was because this felt like a fine knob for control over the innards of analysis that would rarely need to be meddled with. With a properly calibrated size limit, users would typically only need to increase the file limit (instead of decrease). This would mean that they'd need to enter very large numbers of at least 6 digits without error in a command, which seemed unwieldy. If they wanted to quickly lift the size limit, they could use the `-I` flag over cli.